LayoutTests/imported/w3c/web-platform-tests/css/css-tables/animations/auto-layout-calc-width-001.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
```

Comment:
This is in the wrong directory, it should be in LayoutTests/imported/w3c/web-platform-tests/css/css-tables/ instead of LayoutTests/imported/w3c/web-platform-tests/css/css-tables/animations/

That's where it is in the upstream repository at least: https://wpt.fyi/results/css/css-tables/auto-layout-calc-width-001.html?label=experimental&label=master&aligned

Reply:
Yes - I noticed and mentioned to Alan on Slack 1-1 that I will fix it once it will run through EWS so I can fix everything in one push. :-)

Reply:
Fixed in next push. :-)